communicator/questions.go

```diff
@@ -1,19 +1,40 @@
 package main
 
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"log"
+	"math/rand"
+)
+
+const (
+	QUESTIONS_FILE = "assets/questions.json"
+)
+
+type Questions []*Question
+
+var questions Questions
+
 func GetQuestion() (*Question, error) {
```

Comment:
If you return a pointer return a copy. We want to have the question list immutable.